pkg/virtual/workspaces/auth/cache.go

```diff
@@ -181,6 +181,9 @@ type AuthorizationCache struct {
 	syncHandler func(request *reviewRequest, userSubjectRecordStore cache.Store, groupSubjectRecordStore cache.Store, reviewRecordStore cache.Store) error
 
 	rwMutex sync.RWMutex
+
+	watchers    []CacheWatcher
+	watcherLock sync.Mutex
```

Comment:
nit: lock above

Reply:
not sure about the usage pattern, any value of switching to RWMutex?

Reply:
nit fixed in commeit https://github.com/kcp-dev/kcp/pull/613/commits/ffccd523b709550d12f972a5292fc03bf7e76251